IFC4x3/Properties/e/EquipmentDiameter_3V8MuRReb9K8uXog2uN4dH/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="EquipmentDiameter_3V8MuRReb9K8uXog2uN4dH" Name="EquipmentDiameter" UniqueId="df216e1b-6e89-4950-8e21-caa0b85c49d1" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
This name is quite ambiguous. Not sure how to improve it though.

Reply:
Ping @JontyDunn and @hessf-ilf